README.md

```diff
@@ -24,7 +24,14 @@ npm install karma-jasmine-html-reporter --save-dev
 // karma.conf.js
 module.exports = function(config) {
   config.set({
-    reporters: ['kjhtml']
+
+    reporters: ['kjhtml'],
+
+    jasmineHtmlReporter: {
+      // Suppress failed messages (e.g. in combination with karma-mocha-reporter)
+      suppressFailed: true
```

Comment:
I fear this will be copy/pasted even for people not using karma-mocha-reporter. Can you add a 2nd example specifically for karma-mocha-reporter to avoid this? Also, your `reporters` list doesn't actually list karma-mocha-reporter

Reply:
Yes you are correct, will do.